doc/Writerside/topics/firstSteps.md

```
@@ -0,0 +1,8 @@
+# Начало работы
+
+Подсистема интеграции - мощный инструмент для быстрого создания интеграций систем на платформе 1С:Предприятие, с произвольными информационными системами.
+
+## Перед тем как начать
+
+- Пройдите процесс [установки](setup.md) подсистемы в конфигурацию.
+- Выполните [первоначальную настройку](preconfig.md).
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)